### PR TITLE
Issue #1 Add SimpleTrader.Domain Project

### LINKED ISSUE #1: Add Initial Project Domain to Solution
Adds a new project containing the initial domain entities/models and services for CRUD operations

### PR DESCRIPTION
**Pull request by NiiBotchway**
Adds Domain Models such as Account, AssetTransaction, Stock and User Models. Also Adds IDataService for CRUD operations

Closes #1

### CODE REVIEW

**Review by James-Aidoo — COMMENTED** [review]
Change target (base) branch of the PR to `develop` instead of `main`

File: SimpleTrader.Domain/Models/AssetTransaction.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    public class AssetTransaction
+    {
+        public int Id { get; set; }
+        public Account Account { get; set; }
+        public bool IsPurchase { get; set; }
+        public Stock Stock { get; set; }
+        public int Shares { get; set; }
+        public DateTime DateProcessed { get; set; }
+    }
+}
```
Comment (James-Aidoo) [inline SimpleTrader.Domain/Models/AssetTransaction.cs]: Add a new line at end of file

File: SimpleTrader.Domain/Models/Stock.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace SimpleTrader.Domain.Models
+{
+    public class Stock
+    {
+        public string Symbol { get; set; }
+        public double PriceShares { get; set; }
+    }
+}
```
Comment (James-Aidoo) [inline SimpleTrader.Domain/Models/Stock.cs]: Add a new line at end of file

**Review by James-Aidoo — CHANGES_REQUESTED** [review]

**Review by James-Aidoo — APPROVED** [review]

**Comment by NiiBotchway** [comment]
testing comments under PR

**Comment by James-Aidoo** [comment]
> testing comments under PR

Delete this 🤧

**Pull request merged**